jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml

```diff
@@ -22,7 +22,8 @@ spec:
     from:
       kind: DockerImage
       # in a downstream build "latest" will be replaced
-      name: quay.io/modh/odh-minimal-notebook-container:latest
+      # name: odh-minimal-notebook
+      name: $(image-sha-256-odh-minimal-notebook)
```

Comment:
In 1.19 the minimal notebook won't be a dynamic image anymore, we don't need this image replacement.

See this PR https://github.com/red-hat-data-services/odh-manifests/pull/248

Reply:
In order to test the images with rhods live builder, how can we set the image url for each component?